core/src/test/scala/unit/kafka/server/EventBasedFetcherTest.scala

```diff
@@ -0,0 +1,60 @@
+package unit.kafka.server
+
+import kafka.api.IntegrationTestHarness
+import org.apache.kafka.clients.producer.{KafkaProducer, ProducerConfig, ProducerRecord}
+import org.apache.kafka.common.TopicPartition
+import org.apache.kafka.common.serialization.ByteArraySerializer
+import org.junit.Test
+import java.util.Properties
+
+import kafka.utils.TestUtils
+
+class EventBasedFetcherTest extends IntegrationTestHarness{
```

Comment:
is there anything special in this test that covered the change in this patch?  If so, could you add some comments? Do we have other existing tests that cover the basic send function?

Reply:
No, there isn't anything special. So I'll simply remove this test.